src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="course-info-bar course-info-bar--participation" *ngIf="exercise">
+    <div class="row">
+        <div class="col-12 col-sm-12 col-lg-5 d-flex flex-column justify-content-center">
+            <h4>
+                <ng-content select="[pagetitle]"></ng-content>
+            </h4>
+            <div class="d-flex align-items-center">
+                <h6 class="mr-2 mb-0">{{ 'artemisApp.courseOverview.exerciseDetails.dueDate' | translate }}</h6>
+                <span>{{ (exercise.dueDate | date: 'dd/MM/yy HH:mm') || 'N/A' }}</span>
+                <span *ngIf="exercise.dueDate" [ngClass]="exerciseStatusBadge" class="badge ml-2">{{ exercise.dueDate | amTimeAgo }}</span>
+            </div>
+            <div *ngIf="exercise.categories" class="d-sm-flex">
+                <h6 class="mr-1">{{ 'artemisApp.courseOverview.exerciseDetails.categories' | translate }}</h6>
+                <h6 *ngFor="let category of exerciseCategories">
+                    <span class="badge text-white mr-1" [ngStyle]="{ backgroundColor: category.color }">{{ category.category }}</span>
+                </h6>
+            </div>
+        </div>
+        <div class="col-12 col-sm-auto flex-sm-shrink-0 col-lg-3 mt-4 mt-lg-0 d-flex flex-column justify-content-center align-items-start text-nowrap">
+            <table class="exercise-details-table">
+                <tr *ngIf="exercise.maxScore">
+                    <td>
+                        <h6>{{ 'artemisApp.courseOverview.exerciseDetails.maxPoints' | translate }}</h6>
+                    </td>
+                    <td>
+                        <span>{{ exercise.maxScore }} pts.</span>
+                        <jhi-difficulty-badge *ngIf="exercise.difficulty" class="ml-1" [exercise]="exercise"></jhi-difficulty-badge>
+                    </td>
+                </tr>
+                <tr *ngIf="exercise.assessmentDueDate">
+                    <td>
+                        <h6>{{ 'artemisApp.exercise.assessmentDueDate' | translate }}:</h6>
+                    </td>
+                    <td>
+                        <span>{{ (exercise.assessmentDueDate | date: 'dd/MM/yy HH:mm') || 'N/A' }}</span>
```

Comment:
not really related to this PR, but the date format is rather uncommon for Germany and would also be used differently in the US.

Can we consistently use `dd.MM.yy HH:mm` in all places instead of `dd/MM/yy HH:mm`?
This means, can you please run a search and replace
`dd/MM/yy HH:mm` --> `dd.MM.yy HH:mm`

Reply:
I agree that `dd/MM/yy` is pretty confusing. I replaced it.